chore: update bootnode ENR with correct IPV6

chiemerieezechukwu opened the pull request:
**Motivation**

<!-- Why does this PR exist? What are the goals of the pull request? -->

This PR is to fix https://github.com/eth-clients/mainnet/issues/13

Already created this as well https://github.com/eth-clients/mainnet/pull/14

<!-- A clear and concise general description of the changes of this pull request. -->

<!-- If applicable, add screenshots to help explain your solution -->

<!-- Link to issues: Resolves #111, Resolves #222 -->

Closes #issue_number

**AI Assistance Disclosure**

- [ ] External Contributors: I have read the [contributor guidelines](https://github.com/ChainSafe/lodestar/blob/unstable/CONTRIBUTING.md#ai-assistance-notice) and disclosed my usage of AI below.

<!-- Insert any AI assistance disclosure here -->

chiemerieezechukwu committed:
Update bootnode ENR with correct IPV6

philknows commented on packages/cli/src/networks/mainnet.ts:
```diff
@@ -29,5 +29,5 @@ export const bootEnrs = [
   // Lodestar team's bootnodes
   "enr:-IS4QPi-onjNsT5xAIAenhCGTDl4z-4UOR25Uq-3TmG4V3kwB9ljLTb_Kp1wdjHNj-H8VVLRBSSWVZo3GUe3z6k0E-IBgmlkgnY0gmlwhKB3_qGJc2VjcDI1NmsxoQMvAfgB4cJXvvXeM6WbCG86CstbSxbQBSGx31FAwVtOTYN1ZHCCIyg",
-  "enr:-KG4QCb8NC3gEM3I0okStV5BPX7Bg6ZXTYCzzbYyEXUPGcZtHmvQtiJH4C4F2jG7azTcb9pN3JlgpfxAnRVFzJ3-LykBgmlkgnY0gmlwhFPlR9KDaXA2kP6AAAAAAAAAAlBW__4my5iJc2VjcDI1NmsxoQLdUv9Eo9sxCt0tc_CheLOWnX59yHJtkBSOL7kpxdJ6GYN1ZHCCIyiEdWRwNoIjKA",
+  "enr:-KG4QPUf8-g_jU-KrwzG42AGt0wWM1BTnQxgZXlvCEIfTQ5hSmptkmgmMbRkpOqv6kzb33SlhPHJp7x4rLWWiVq5lSECgmlkgnY0gmlwhFPlR9KDaXA2kCoGxcAJAAAVAAAAAAAAABCJc2VjcDI1NmsxoQLdUv9Eo9sxCt0tc_CheLOWnX59yHJtkBSOL7kpxdJ6GYN1ZHCCIyiEdWRwNoIjKA",
```
Needs to be added in identified file here too @chiemerieezechukwu

chiemerieezechukwu replied:
I updated it as well

nflaig reviewed (commented):
the ENR looks good to me, but should be updating tests as noted by the AI

nflaig commented:
what about the hoodi enr? https://github.com/eth-clients/hoodi/pull/24

chiemerieezechukwu committed:
Update ENR in parseBootnodesFile.test.ts

chiemerieezechukwu committed:
add hoodi bootnode

chiemerieezechukwu commented:
> what about the hoodi enr? [eth-clients/hoodi#24](https://github.com/eth-clients/hoodi/pull/24)

Fixed the test according to the AI and added the Hoodi bootnode ENR as well

nflaig reviewed (approved):
LGTM

wemeetagain commented:
:tada: This PR is included in [v1.39.0](https://github.com/ChainSafe/lodestar/releases/tag/v1.39.0) :tada:
